src/store/modules/ADempiere/dataManager.js

```diff
@@ -0,0 +1,73 @@
+import {
+  requestListEntities
+} from '@/api/ADempiere/common/persistence'
+
+const dataManager = {
+  state: {
+    containerData: []
+  },
+
+  mutations: {
+    setContainerData(state, payload) {
+      const index = state.containerData.findIndex(element => {
+        return element.containerUuid === payload.containerUuid
+      })
+      if (index > -1) {
+        // update records
+        state.containerData.splice(index, 0, payload)
+      } else {
+        state.containerData.push(payload)
+      }
+    }
+  },
+
+  actions: {
+    getListEntities({
+      commit
+    }, {
+      parentUuid,
+      containerUuid,
+      tableName,
+      query,
```

Comment:
remove this line

Reply:
without the query how will the query work?